beacon-chain/builder/service.go

```diff
@@ -66,7 +65,7 @@ func NewService(ctx context.Context, opts ...Option) (*Service, error) {
 
 		// Is the builder up?
 		if err := s.c.Status(ctx); err != nil {
-			return nil, fmt.Errorf("could not connect to builder: %v", err)
+			log.WithError(err).Error("Failed to check builder status")
```

Comment:
the next line says "builder has been configured" maybe we should put this in an else statement or return early here.

Reply:
+1, thats my only feedback as well